torch/lib/libshm/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(libshm C CXX)
-cmake_minimum_required(VERSION 2.6 FATAL_ERROR)
-cmake_policy(VERSION 2.6)
+cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
```

Comment:
I dunno why we dont just bump to the minimum requirement of PyTorch main project. which is already 3.5?

Reply:
don't know for sure, but it's possible people can import pytorch's internal as libraries, and they may maintain their own cmake version. keeping the lowest viable version gives the max compatibility I guess

Reply:
Please update to 3.5

Reply:
oh, "shipit" was in queue before the comment, do you think we can bump up the min version for all `cmake_minimum_required` within pytorch repo? I can followup with a PR to change it